types/request/index.d.ts

```diff
@@ -125,7 +125,7 @@ declare namespace request {
         oauth?: OAuthOptions | undefined;
         aws?: AWSOptions | undefined;
         hawk?: HawkOptions | undefined;
-        qs?: any;
+        qs?: { [k: string]: any };
```

Comment:
Unless I'm mistaken, this type should turn out to be identical to that of `Request.q` below, so `object`?

Reply:
[jakebailey](https://github.com/jakebailey) you're right, they should match. But `object` is incorrect. 
`object` represents all values that are non-primitive and is generally, as is in this case, misused. 
A "plain object" should be denoted as `{ [k: string]: any }` or its alias `Record<string, any>`

[hishtadlut](https://github.com/hishtadlut) could you update line 230 too, to match please.

Reply:
I can't think of any examples off the top of my head that `object` would fail at, but I think the alternatives are clearer for sure.